packages/SwingSet/src/kernel/vatManager/nodeWorker.js

```diff
@@ -135,9 +137,16 @@ export function makeNodeWorkerVatManagerFactory(tools) {
       return pr.promise;
     }
 
-    function replayTranscript() {
-      // TODO unimplemented
-      throw Error(`replayTranscript not yet implemented`);
+    async function replayTranscript() {
```

Comment:
This `replayTranscript` implementation looks plausible and doesn't blow up in light testing, but I'm not qualified to judge whether it's correct. I trust @FUDCo will review more closely.